app-config.yaml

```diff
@@ -274,6 +274,13 @@ scaffolder:
         $env: BITBUCKET_TOKEN
     visibility: public # or or 'private'
 
+chromeUXReport:
+  projectId: 'example'
+  keyPath: 'example.json'
+  origins:
+    - site: 'https://www.example.com'
+      name: 'Example'
```

Comment:
Probably leave this commented out as an example instead, although I'm not quite sure what the result of doing that is just yet :D

Reply:
I wrote this as an example. This config can be any site url. I updated in a more understandable way :)